### ISSUE TITLE
tensorflow on BeagleBone X15

### ISSUE THREAD

**Issue by yuanbuckingham** [samjabrahams/tensorflow-on-raspberry-pi#136]
### Describe the Issue
I succesfully installed tensorflow from soucre on BBB according to the tutorial https://github.com/samjabrahams/tensorflow-on-raspberry-pi/blob/master/GUIDE.md.
I use pre-trained DL model to detect objects on BBB, it runs well but just too slow. So I decided to use BeagleBoard X15 (BBX15) which has much more processor than BBB. I followed the same steps for raspberry pi, but just didn't add swap space, because BBX15 has enough memory. So I installed Bazel 0.8.0 successfully, but when I start to build tensorflow, it gives me the errors as follows:

WARNING: /root/.cache/bazel/_bazel_root/6633267185f34e44f59a791ee98f07b8/externa
l/protobuf_archive/WORKSPACE:1: Workspace name in /root/.cache/bazel/_bazel_root
/6633267185f34e44f59a791ee98f07b8/external/protobuf_archive/WORKSPACE (@com_goog
le_protobuf) does not match the name given in the repository's definition (@prot
obuf_archive); this will cause a build error in future versions
ERROR: /home/debian/tf/tensorflow/tensorflow/BUILD:399:12: Label '//tensorflow:t
ools/integration_tests/gcs_smoke_test/gcs_smoke.py' crosses boundary of subpacka
ge 'tensorflow/tools/integration_tests/gcs_smoke_test' (perhaps you meant to put
 the colon here: '//tensorflow/tools/integration_tests/gcs_smoke_test:gcs_smoke.
py'?)
ERROR: /home/debian/tf/tensorflow/tensorflow/BUILD:399:12: Label '//tensorflow:t
ools/integration_tests/gcs_smoke_test/setup.sh' crosses boundary of subpackage '
tensorflow/tools/integration_tests/gcs_smoke_test' (perhaps you meant to put the
 colon here: '//tensorflow/tools/integration_tests/gcs_smoke_test:setup.sh'?)
ERROR: /home/debian/tf/tensorflow/tensorflow/BUILD:399:12: Label '//tensorflow:t
ools/integration_tests/gcs_smoke_test/BUILD.bazel' crosses boundary of subpackag
e 'tensorflow/tools/integration_tests/gcs_smoke_test' (perhaps you meant to put 
the colon here: '//tensorflow/tools/integration_tests/gcs_smoke_test:BUILD.bazel
'?)                                                                             
ERROR: /home/debian/tf/tensorflow/tensorflow/BUILD:399:12: Label '//tensorflow:t
ools/integration_tests/gcs_smoke_test/teardown.sh' crosses boundary of subpackag
e 'tensorflow/tools/integration_tests/gcs_smoke_test' (perhaps you meant to put 
the colon here: '//tensorflow/tools/integration_tests/gcs_smoke_test:teardown.sh
'?)                                                                             
ERROR: /home/debian/tf/tensorflow/tensorflow/BUILD:399:12: Label '//tensorflow:t
ools/integration_tests/gcs_smoke_test/test_wrapper.sh' crosses boundary of subpa
ckage 'tensorflow/tools/integration_tests/gcs_smoke_test' (perhaps you meant to 
put the colon here: '//tensorflow/tools/integration_tests/gcs_smoke_test:test_wr
apper.sh'?)                                                                     
ERROR: /home/debian/tf/tensorflow/tensorflow/tools/pip_package/BUILD:134:1: Targ
et '//tensorflow:windows' contains an error and its package is in error and refe
renced by '//tensorflow/tools/pip_package:build_pip_package'                    
ERROR: /home/debian/tf/tensorflow/tensorflow/tools/pip_package/BUILD:134:1: Targ
et '//tensorflow:windows_msvc' contains an error and its package is in error and
 referenced by '//tensorflow/tools/pip_package:build_pip_package'               
ERROR: Analysis of target '//tensorflow/tools/pip_package:build_pip_package' fai
led; build aborted: Loading failed
INFO: Elapsed time: 0.508s
FAILED: Build did NOT complete successfully (0 packages loaded)
### Steps to Reproduce
bazel build -c opt --copt="-mfpu=neon-vfpv4" --copt="-funsafe-math-optimizations" --copt="-ftree-vectorize" --copt="-fomit-frame-pointer" --local_resources 1024,1.0,1.0 --verbose_failures tensorflow/tools/pip_package:build_pip_package

### Hardware/Software Info

Processor: TI AM5728 2×1.5-GHz ARM® Cortex-A15
2GB DDR3 RAM
4GB 8-bit eMMC on-board flash storage
2D/3D graphics and video accelerators (GPUs)
2×700-MHz C66 digital signal processors (DSPs)
2×ARM Cortex-M4 microcontrollers (MCUs)
4×32-bit programmable real-time units (PRUs)

* Operating System used: Debian
* Version of Python used: 2.7.13
* SD card memory size: 32G
* TensorFlow git commit hash (if building from source): 

### Relevant Console Output/Logs
